packages/components/theme/src/components/avatar.ts

```diff
@@ -35,7 +35,9 @@ const baseStyleContainer = defineStyle((props) => {
   const borderColor = mode("white", "gray.800")(props)
 
   return {
-    bg,
+    "&:not([data-loaded])": {
+      bg,
+    },
```

Comment:
Please use a css-var here. In case a user is already overriding the bg from the theme.

Assign the value to a CSS var and use `bg: $avatarBg.reference` at the root.